### DISCUSSION TITLE
I hope to add imported txt song list

### DISCUSSION THREAD

**Discussion by wyllwyc** [navidrome/navidrome#2665] [Ideas]
I hope to add imported txt song list

**Comment by deluan** [comment]
Can you clarify what exaclty you are trying to accomplish? You want to import `.m3u` files? Just drop them in the same place as your music and they should be imported. Make sure the paths inside the `.m3u` are valid.

**Reply by wyllwyc** [reply]
There are websites that can export Spotify playlists to txt. I hope that the playlist can be generated by importing txt.

**Reply by deluan** [reply]
Navidrome does not support txt playlists, but there are some tools out there that creates `.mp3` files from Spotify playlists.

**Reply by wyllwyc** [reply]
Thank you.
